src/maintainer/adding_pkgs.rst

```diff
@@ -111,6 +113,54 @@ The maintainer's job is to:
 - Answer eventual questions about the package on the feedstock issue tracker.
 
 
+Adding multiple packages at once
+--------------------------------
+
+If you would like to add more than one related packages, they can be added to staged-recipes in a single pull request (in separate directories). If the packages are interdependent (i.e. one package being added lists one or more of the other packages being added as a requirement), conda-build will be able to locate the dependencies that are only present within staged-recipes as long as the builds finish in the dependencies order. Using a single pull request cuts allows you to quickly get packages set up without waiting for each package in a dependency chain to be reviewed, built, and added to the conda-forge channel before starting the process over with the next recipe in the chain.
+
+.. note::
+
+   There will be a race condition if the packages are interdependent such that there will an error if a build finishes before its dependency is built.
```

Comment:
```suggestion
   There will be a race condition if the packages are interdependent since feedstocks for all the packages are created at the same time and builds will start immediately without waiting for dependencies.
```

Reply:
Can you all add instructions on how to restart the builds when the race condition happens?

Reply:
I am not familiar with that process, but if someone can give me a summary, I'm happy to add it to the docs.

Reply:
Thanks for doing this.

Here are the instructions, https://gitter.im/conda-forge/conda-forge.github.io?at=5d23549b3b186e65696adea8

For eg: See click the red crosses or green check marks at https://github.com/conda-forge/numpy-feedstock/commits/master

Reply:
Azure is a new product and the restarting button is not really working.
However, there is someone from Microsoft here working on that.

For now we have the workaround of pushing an empty commit from a new PR with:

```
git commit --amend --no-edit && git push --force
```

Reply:
I updated the note to suggest this command.